src/inc/test/CommonState.hpp

```diff
@@ -303,7 +303,7 @@ class CommonState
         const CONSOLE_INFORMATION& gci = Microsoft::Console::Interactivity::ServiceLocator::LocateGlobals().getConsoleInformation();
         // length 80 string of text with bisecting characters at the beginning and end.
         // positions of き(\x304d) are at 0, 27-28, 39-40, 67-68, 79
-        PWCHAR pwszText =
+        auto pwszText =

```

Comment:
interesting. not necessary for this change, i presume, but I'll allow it? 😁

Reply:
Similar to the ClampSub issue above, idk why this came up:
```
6>D:\Terminal\src\inc\test\CommonState.hpp(306,25): error C2440: 'initializing': cannot convert from 'const wchar_t [81]' to 'PWCHAR'
6>D:\Terminal\src\inc\test\CommonState.hpp(306,25): message : Conversion from string literal loses const qualifier (see /Zc:strictStrings)
```
🤷‍♂️